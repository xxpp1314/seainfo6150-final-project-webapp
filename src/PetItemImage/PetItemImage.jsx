import React from "react";
import PropTypes from "prop-types";
import styles from "./PetItemImage.module.css";

const PetItemImage = (props) => {
  return (
    <div >
      <img className={styles.image} 
          src={props.url} alt="">
      </img>
    </div>
    
  );
};

PetItemImage.propTypes = {
  url: PropTypes.string.isRequired,
};

export default PetItemImage;