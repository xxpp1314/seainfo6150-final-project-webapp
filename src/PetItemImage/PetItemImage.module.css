.image{
    padding: 2%;
    width: 80%;
    height:250px;
    border-radius: 10%;  
}