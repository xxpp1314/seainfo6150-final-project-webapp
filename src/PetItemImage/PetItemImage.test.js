import React from "react";
import { render } from "@testing-library/react";
import PetItemImage from "./PetItemImage.jsx";

it("renders the PetItemImage correctly", () => {
  const { container } = render(
    <PetItemImage
      url="https://dl5zpyw5k3jeb.cloudfront.net/photos/pets/51169344/1/?bust=1619142914&width=1080"
    />
  );
  expect(container).toMatchSnapshot();
});