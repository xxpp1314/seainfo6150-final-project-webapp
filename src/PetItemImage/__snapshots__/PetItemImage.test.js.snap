// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the PetItemImage correctly 1`] = `
<div>
  <div>
    <img
      alt=""
      class="image"
      src="https://dl5zpyw5k3jeb.cloudfront.net/photos/pets/51169344/1/?bust=1619142914&width=1080"
    />
  </div>
</div>
`;
